tests/integration/pilot/common/yaml.go

```diff
@@ -0,0 +1,38 @@
+//go:build integ
+// +build integ
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import "istio.io/istio/pkg/test/framework"
+
+// ApplyFileOrFail applies the given yaml file and deletes it during context cleanup
+func ApplyFileOrFail(t framework.TestContext, ns, filename string) {
```

Comment:
Replace usages with `ctx.ConfigIstio().File(meshNs.Name(), path.Join(env.IstioSrc, "samples/sleep/sleep.yaml")).ApplyOrFail(ctx)` and remove this please. We do not need more ways to call the same code

Reply:
Yes, sure. I removed this file. I didn't find `ctx.ConfigIstio().File()` at the beginning and I copied this function from analyze_test.go, because I wanted to move it to common package, reuse and then remove from the original location.